src/modes/ModeEstimation.jl

```diff
@@ -147,6 +147,50 @@ function (f::OptimLogDensity)(z)
     return -DynamicPPL.getlogp(varinfo)
 end
 
+function (f::OptimLogDensity)(F, G, z)
```

Comment:
I'm not sure if it's useful to keep this separate definition? It seems we only need `f(F, G, H, z)`, so the implementation could just be included there directly.